testsuite/model/pom.xml

```diff
@@ -35,6 +35,12 @@
             <artifactId>junit</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>net.jcip</groupId>
+            <artifactId>jcip-annotations</artifactId>
```

Comment:
The version is the same as the transitive dependency in `resteasy-jaxrs`. But not sure if we actually need or want to have the properties in the parent pom.

Reply:
Is it used in model tests at all? Is it necessary to be added?

Reply:
That is what I asking but looks like you don't know either. Thanks.

Reply:
I got response offline:

> Yeah, there is a test using an annotation
> @Guard or something
> org.infinispan.persistence.manager.PersistenceManagerImpl

So I'm fine for this PR. See #10924 for revert requirement

Reply:
Tks